packages/terra-embedded-content-consumer/tests/nightwatch/embedded-content-consumer-spec.js

```diff
@@ -2,7 +2,7 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
 
-module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
+module.exports = resizeTo(['tiny', 'medium', 'huge'], {
```

Comment:
this probably only needs one breakpoint. All it does is mount XFC and doesn't need any responsive tests.

Reply:
Updated: https://github.com/cerner/terra-core/pull/1166/commits/3f3d3d93af165fad40df984045b5a2425b87fc8b